liquidctl/driver/nzxt_smart_device.py

```diff
@@ -45,11 +45,13 @@
 PWM and DC control is supported, and the device automatically chooses the appropriate
 mode for each channel.
 
-Additionally, it features two independent lighting (Addressable RGB) channels,
-unlike the single channel in the original. NZXT Aer RGB 2 fans and HUE 2 lighting
-accessories (HUE 2 LED strip, HUE 2 Unerglow, HUE 2 Cable Comb) can be
-connected. The firmware installed on the device exposes several color presets, most
-of them common to other NZXT products.
+Additionally, it features two lighting (Addressable RGB) channels, `led1` and `led2`.
+Color modes can be set independently for each lighting channel, but the specified 
+color mode will then apply to all devices daisy chained on that channel.
+
+NZXT Aer RGB 2 fans and HUE 2 lighting accessories (HUE 2 LED strip, HUE 2 Unerglow,
+HUE 2 Cable Comb) can be connected. The firmware installed on the device exposes
+several color presets, most of them common to other NZXT products.
```

Comment:
This docstring was fine and the beginning of the `.md` file were fine.

The new sentence should go/replace what's in the "RGB lighting" section of the `.md` doc file.

Reply:
Actually, I'll make this change and review the docs one last time for you.

This is something I can do.  You focus on those last _functional_ checks.